scripts/ensure-rebar3.sh

```diff
@@ -3,14 +3,19 @@
 set -euo pipefail
 
 VERSION="$1"
+REBAR3_FILENAME="${REBAR3_FILENAME:-rebar3}"
 
 # ensure dir
 cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")/.."
 
 DOWNLOAD_URL='https://github.com/emqx/rebar3/releases/download'
 
 download() {
-    curl -f -L "${DOWNLOAD_URL}/${VERSION}/rebar3" -o ./rebar3
+    curl -f -L "${DOWNLOAD_URL}/${VERSION}/${REBAR3_FILENAME}" -o ./rebar3
```

Comment:
download old version (`3.14.3-emqx-8`) for otp 23 and new version for 24 ?

Reply:
OK, I'll change this.